### ISSUE TITLE
how do I get the API_Key?

### ISSUE THREAD

**Issue by modernNeo** [BestBuyAPIs/bestbuy-sdk-js#55]
I see that [Getting Started](https://github.com/BestBuyAPIs/bestbuy-sdk-js#getting-started) tells me to go to  https://developer.bestbuy.com/ and create an account there but I don't seem able to do that.

All I see is
> Free email and .edu addresses are not allowed at this time.

But that doesn't make sense given the open source nature of the libraries...

Can I get help with this?

**Comment by albinsony95** [comment]
I am having that same issue. If you figure it out why, could you please quote to this comment reply.

**Comment by modernNeo** [comment]
the fact that it's been almost 2 years and they still haven't responded makes me think that this repo is not maintained, especially since the last commit was 3 years ago.

**Comment by albinsony95** [comment]
I am looking for a similar API for a capstone project. Do you know any other similar APIs which I could use for Digital shop website?

**Comment by modernNeo** [comment]
honestly, the only solutions I can think for you verges on just scraping the rendered html 🤷🏿‍♀️ something selenium is good for.

**Comment by albinsony95** [comment]
I was thinking of a JSON file for products for my backend which display on the frontend. I am not sure how to render html using selenium. I will give it a try . thanks

**Comment by modernNeo** [comment]
there's lots of tutorials online for that kinda thing 😄 even easier if you are using python.

**Comment by albinsony95** [comment]
I am using node and express server as backend for the Capstone which I learned in bootcamp. react as frontend. not sure of python. need to learn from scrach.

**Comment by modernNeo** [comment]
if you don't mind using the Canadian site, you should try https://www.bestbuy.ca/en-ca. the Canadian version uses JSONs.

open the network console and take a look at how the JSONs are requested.

**Comment by Flet** [comment]
Some of the reasoning is explained in this old Medium post:
https://medium.com/best-buy-developers/announcing-a-change-to-best-buy-s-api-access-b09afc4bc27a

TLDR: need to use a non-freemail address.

However, registering a new domain and setting it to forward to a freemail account will work though!

**Comment by modernNeo** [comment]
and God forbid this be documented anywhere 🤦🏿‍♀️ 

but nevertheless, thanks for the info @Flet

**Comment by Flet** [comment]
> I am looking for a similar API for a capstone project. Do you know any other similar APIs which I could use for Digital shop website?

@albinsony95 Not sure if this helps, but check out https://github.com/BestBuy/api-playground

Its pretty old and dusty, but it might be worth looking at:
>The Best Buy API Playground is an API training tool for students, educators and other learners to explore the possibilities of a fully functional [RESTful API](http://www.restapitutorial.com/) in a simple, non-production environment. API Playground was developed by Best Buy and makes use of a [Creative Commons licensed](https://creativecommons.org/licenses/by-nc/4.0/) dataset including over 50,000 products and store-related information - making it easy for teaching APIs (both consumption and creation) to classrooms with realistic data. The playground supports full [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) (Create Read Update Delete) operations for all API endpoints and does not require any external services nor databases.

and if you're just looking for a big set of realistic data, there is a data set here (the playground uses this one I believe)
https://github.com/BestBuyAPIs/open-data-set

good luck!
